lib/utils/AuthService.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:water_overflow/models/User.dart' as myUser;
+
+class AuthService {
+  final FirebaseAuth _fAuth = FirebaseAuth.instance;
```

Comment:
This value must be private if we don't use it somewhere outside of our class

Reply:
> This value

class or variable?

Reply:
The variable that is using only inside current class